zen/ai/mobile_adapter.py

```
@@ -169,141 +177,140 @@ def process(self, input_text: str, context: MobileContext) -> str:
 echo "📱 Enhanced mobile response"
 echo "🌉 airi-zenOS bridge active"
 """
-            
-            with tempfile.NamedTemporaryFile(mode='w', suffix='.sh', delete=False) as f:
+
+            with tempfile.NamedTemporaryFile(mode="w", suffix=".sh", delete=False) as f:
                 f.write(script_content)
                 script_path = f.name
-            
+
             os.chmod(script_path, 0o755)
```

Ruff (0.16.0)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)